### PR TITLE
Mandale cumbia

### PR DESCRIPTION
**Pull request by LisandroTorresetti**

### CODE REVIEW

**Review by IgnacioCarol — APPROVED** [review]

File: resources/config.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@
         </niveles>
     </escenario>
     <credenciales>
-         <cantidadJugadores>2</cantidadJugadores>
+         <cantidadJugadores>1</cantidadJugadores>
```
Comment (IgnacioCarol) [inline resources/config.xml]: Try to not change these values, the default should be 2 (it's okay for now, the next person will return it to 2 but for the next branch)

File: src/CharacterStates/CharacterState.cpp [hunk 2]
```diff
@@ -6,6 +6,10 @@ CharacterState::CharacterState(int currentFrame, int framesAmount) {
     this -> framesAmount = framesAmount;
 }
 
+CharacterState::CharacterState() {
+
```
Comment (IgnacioCarol) [inline src/CharacterStates/CharacterState.cpp:10]: Why is this necessary? (It is not necessary to be changed, but I do not know what purpose this accomplish)

File: src/CharacterStates/CharacterState.h [hunk 3]
```diff
@@ -23,15 +26,20 @@ class CharacterState {
 
     virtual void changeState(const Uint8* currentKeyStates, Player* player);
 
-    int getCurrentFrame();
     int getFramesAmount();
 
     std::string getStateType();
 
 protected:
+    static const int NORMAL_FRAME = 0;
```
Comment (IgnacioCarol) [inline src/CharacterStates/CharacterState.h]: If every state will use a different kind of frame, wouldn't it be better if we simple make FRAME_AMOUNTS and each of the following states replace that value?. Ex: jumping state will use the frame amount 4 and only will have to override the value of FRAME_AMOUNTS instead of using the rest of the frames amounts

File: src/Client/GameClient.cpp [hunk 4]
```diff
@@ -154,6 +161,16 @@ bool GameClient::createGameObjects(GameObjectsInit gameObjectsInit) {
     return true;
 }
 
+void GameClient::loadSounds(std::map<std::string, std::string> soundPaths) {
+    for (std::pair<std::string, std::string> element : soundPaths){
+        musicManager->addPath(element.first, element.second, false);
+    }
+
+    musicManager->addPath(MUSIC, "Sound_Effects/Music/SuperMarioBrosSong.mp3", true);
```
Comment (IgnacioCarol) [inline src/Client/GameClient.cpp]: Please check if in linux .wav works as well as with mp3, in mac only wav works ok

File: src/Client/GameClient.cpp [hunk 5]
```diff
@@ -299,3 +320,23 @@ void GameClient::changeLevel(GameMsgLevelChange nextLevelConfig) {
 void GameClient::setServerDown() {
     serverIsDown = true;
 }
+
+void GameClient::pauseSoundEffects(int music, int sounds) {
+    if (music && !musicManager->isMusicPaused()){
```
Comment (IgnacioCarol) [inline src/Client/GameClient.cpp]: I don't really like the double check, maybe it would be more performant (or nice) doing smt like this: 
```suggestion
    if (music) {
        if (musicManager->isMusicPaused()) {...}
        else {...}
```

Comment (LisandroTorresetti) [reply src/Client/GameClient.cpp]: Te lo cambio, pero no digas performante porque es UN IF hdp, es O(1) de aca a la china en cualquier caso
